contracts/ERC20WithPermit.sol

```diff
@@ -12,9 +12,9 @@
 
 // SPDX-License-Identifier: MIT
 
-pragma solidity <0.8.0;
+pragma solidity 0.8.4;
 
-import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
```

Comment:
I `SafeMath` still needed in `ERC20WithPermit.sol`?
I see that `add()` is still used, but it could probably be replaced with `+`.